### ISSUE TITLE
Error while installing new version of mblab and while creating new male characters

### ISSUE THREAD

**Issue by HarshEcka** [animate1978/MB-Lab#295]
While installing addon i received error in both windows and linux. screenshot attached. And while creating male characters again error was displayed (Screenshot [attached).
![IMG_20200801_215150](https://user-images.githubusercontent.com/69085843/89106286-5067ca80-d446-11ea-9c71-ea5f4a51181a.jpg)

![IMG_20200801_215256](https://user-images.githubusercontent.com/69085843/89106293-5c538c80-d446-11ea-9ebb-098ab1513f11.jpg)
![IMG_20200801_215324](https://user-images.githubusercontent.com/69085843/89106295-64abc780-d446-11ea-8c5a-e44d8afe4bf0.jpg)

 I used blender version 2.83.3 and 2.90 beta. I was using the beta version of the addon till today evening.

**Comment by animate1978** [comment]
I think I figured it out... the male mesh does not seem to have the shader attached... why I have no clue, plus why would it crash. I am working on fixing this.

**Comment by animate1978** [comment]
I just committed the updated file. Somehow somewhere the skin shader on the human male mesh was missing, this has been fixed. I tried myself and got the same error and thankfully to your images was able to figure out exactly where it would crash.

I also updated the Patreon post about this so that users can download this as well.

This is why I decided to post on Patreon first ... see if I get some bugs and wow I am glad I did because that kind of bug is a big one.
